bundles/org.openhab.binding.touchwand/README.md

```diff
@@ -40,6 +41,32 @@ After adding TouchWand Hub the auto discovery will add all switches dimmers and
 
 No thing configuration is needed
 
+## Channels 
+
+
+note **Touchwand Wanderfull™** supports various types of alarm sensors such as water leak, door/window sensor and motion sensor.
+in order to keep the Alarm Sensor generic and support all AlarmSensor types, All the alarm sensor channels are available for each sensor
+but only the relevant ones will be updates with the relevant values.
+It is the user responsibility to use the proper channels for the sensors.
```

Comment:
Do you know at runtime what channels a sensor actually supports? You can remove channels dynamically if a sensor doesn't support them. I would suggest doing that instead of putting the burden on the user to figure out which ones work.

Reply:
@cpmeister  , I thought about it too but could not find a way to do it .
Is there a documentation on how to add/remove per defined channels  dynamically ?

Reply:
@cpmeister  , ok found a thread https://community.openhab.org/t/dynamically-add-channels-defined-in-xml/103940 , I believe it is a good starting point , please let me know if there are other sources I can use

Reply:
I would just like to add that to remove a channel you would call `thingBuilder.withoutChannel(channel);`

Reply:
done